### PR TITLE
edited nav bar to be collapsible and made it responsive to screen size

### PR DESCRIPTION
**Pull request by TimiOmo**

### CODE REVIEW

**Comment by ganning127** [comment]
Good start! When I scroll, down, the bottom left corner also scrolls. Could we fix that please?

Screenshot 2024-11-05 at 4 32 06 PM

**Comment by TimiOmo** [comment]
The issue should be fixed now!

**Comment by ganning127** [comment]
Looks good to me!

**Comment by smarru** [comment]
Thanks for your contributions @TimiOmo May be @lahirujayathilake can also review it before we merge

**Review by lahirujayathilake — CHANGES_REQUESTED** [review]

File: custos-portal/src/lib/constants.ts [hunk 1]
```diff
@@ -5,15 +5,17 @@ export let CLIENT_ID:string;
...
 
-if (!process.env.NODE_ENV || process.env.NODE_ENV === 'development') {
-    CLIENT_ID = 'veda-dafsxhsztbsczrmmbftw-10000000';
-    BACKEND_URL = 'http://localhost:8081';
+// if (!process.env.NODE_ENV || process.env.NODE_ENV === 'development') {
+//     CLIENT_ID = 'veda-dafsxhsztbsczrmmbftw-10000000';
+//     BACKEND_URL = 'http://localhost:8081';
+//     APP_URL = 'http://localhost:5173'
+// } 
+// else {
```
[the remark this answers is not in the document]
Comment (TimiOmo) [reply custos-portal/src/lib/constants.ts]: Comments should be gone now

**Pull request merged**